react-client-app/src/api/values.js

```diff
@@ -25,6 +25,10 @@ const usePost = async (url,data) =>{
         body: JSON.stringify(data)    
     }
     );
+    if (response.status < 200 || response.status > 300) {
+        console.log(response)
+        throw response
+    }
```

Comment:
What happens to cause the status to be less than 200 or greater than 300? Is this bit just to show you something in the terminal when something goes wrong?

Reply:
If the status is less than 200 or more than 300 then there was an error on the server, and the usePost function throws an error

Reply:
Is it not greater than 500 for server errors? 400-499 is for when there a problem with the information the client is sending, if I'm not mistaken. Is this meant to simply catch when something goes wrong when doing `usePost`?

Reply:
Yeah it's just to catch when there is any error status in usePost